cpp_utils/BLEUtils.cpp

```diff
@@ -1603,10 +1600,9 @@ void BLEUtils::dumpGattServerEvent(
 		} // ESP_GATTS_CREATE_EVT
 
 		case ESP_GATTS_DISCONNECT_EVT: {
-			ESP_LOGD(LOG_TAG, "[conn_id: %d, remote_bda: %s, is_connected: %d]",
+			ESP_LOGD(LOG_TAG, "[conn_id: %d, remote_bda: %s]",
 				evtParam->connect.conn_id,
-				BLEAddress(evtParam->connect.remote_bda).toString().c_str(),
-				evtParam->connect.is_connected);
```

Comment:
Why remove the logging of is_connected?

Reply:
esp-idf removed that field in commit 79f206be47c3f608615c1de8c491107e6c9194bb.